<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Index is saved in:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>About:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Answer:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtIndexPath">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>12</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>User Information:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>52</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Note: Split with an Enter.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtAnswer">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Question:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbQuestion">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>110</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>UserName:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txtMaterial">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>201</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>481</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBroswe">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Broswe</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbUser">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Admin</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>All Materials:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>430</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancle</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPassword">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 28pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>Lazy</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="txtUpdate">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>341</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labUpdate">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Build Time:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labVersion">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>260</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>v0.1-alpha(0100)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>Formula</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUpdate">
   <property name="geometry">
    <rect>
     <x>229</x>
     <y>399</y>
     <width>343</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Check Update</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>277</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkLogin">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>NeedLogin</string>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>progressBar</zorder>
  <zorder>label_13</zorder>
  <zorder>label_2</zorder>
  <zorder>label_10</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>txtIndexPath</zorder>
  <zorder>label_9</zorder>
  <zorder>label_4</zorder>
  <zorder>txtAnswer</zorder>
  <zorder>label_5</zorder>
  <zorder>cbQuestion</zorder>
  <zorder>label_6</zorder>
  <zorder>txtMaterial</zorder>
  <zorder>btnSave</zorder>
  <zorder>btnBroswe</zorder>
  <zorder>cbUser</zorder>
  <zorder>label_3</zorder>
  <zorder>btnCancel</zorder>
  <zorder>txtPassword</zorder>
  <zorder>label</zorder>
  <zorder>txtUpdate</zorder>
  <zorder>labUpdate</zorder>
  <zorder>labVersion</zorder>
  <zorder>btnUpdate</zorder>
  <zorder>chkLogin</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>Settings</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>536</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
